benchmark/modules/run_1yr_fullchem_benchmark.py

```diff
@@ -66,7 +66,7 @@
 warnings.filterwarnings("ignore", category=UserWarning)
 warnings.filterwarnings("ignore", category=FutureWarning)
 
-def run_benchmark(config):
+def run_benchmark(config, bmk_year_ref="2019", bmk_year_dev="2019"):
```

Comment:
Do we need to supply default values for year here if the yaml file will always have start/end? I'm thinking ahead to when we change year since we'll have to remember to update these values, which might not be obvious since they are set in the config file.

Reply:
Good point, I was initially trying to make sure run_1yr_fullchem_benchmark.py was functional as a standalone script, but I think that just adds unnecessary confusion. Removed